deploy-edpm.yml

```diff
@@ -35,6 +35,16 @@
 - name: Import admin setup related playbook
   ansible.builtin.import_playbook: ci_framework/playbooks/07-admin-setup.yml
 
+  # Note(Chandan): Add 5 mins sleep to finish the EDPM deployment
+  # before running tempest to avoid no-valid host.
+  # It is a known issue with dev-preview dataplane operator.
```

Comment:
I would say:
... It is a know issue with the dev-preview nova-operator that it runs the libvirt and nova-compute deployment twice but reports ready status after the first run. So with this sleep the job waits for the deployment to really stabilize before start running the tempest tests.